src/views/Home.jsx

```diff
@@ -18,12 +22,51 @@ export function Home({ listToken, setListToken }) {
 		}
 	}, [listToken]);
 
+	const handleChange = (e) => {
+		setExistingToken(e.target.value);
+	};
+
+	const handleSubmit = (e) => {
+		e.preventDefault();
+
+		// conects to db
+		// checks if the token that user inputted is associated with any lists
+		streamListItems(existingToken, (snapshot) => {
+			const nextData = getItemData(snapshot);
+
+			// if there is a list, set the token to the listToken state
+			if (nextData.length > 0) {
+				setListToken(existingToken);
```

Comment:
And if there happens to be a list, we set the `listToken` state with `caseSensitiveToken` 
```suggestion
				setListToken(caseSensitiveToken);
```